README.md

```diff
@@ -67,3 +67,29 @@ class UserAgentClient extends http.BaseClient {
   }
 }
 ```
+## Using on the Browser
+The HTTP library can be used on the browser via the BrowserClient class in package:http/browser_client.dart. This client translates requests into XMLHttpRequests. For example:
+
+```dart
+import 'dart:async';
+import 'package:http/browser_client.dart';
+
+main() async {
+  var client = new BrowserClient();
```

Comment:
Note that the regular `Client` class from the regular `package:http/http.dart` import should also give you a `BrowserClient`. This is a more portable mechanism so it is probably what we should advertise.

I believe that the `browser_client.dart` is pretty much only exists for historical reasons at this point, although its possible that it has some additional apis on top of the regular Client interface (not sure though).